.golangci.yml

```diff
@@ -0,0 +1,24 @@
+linters-settings:
+  gocyclo:
+    min-complexity: 20
+  gofmt:
+    simplify: true
+  golint:
+    # minimal confidence for issues, default is 0.8
+    min-confidence: 0
+  govet:
+    check-shadowing: true
+  misspell:
+    locale: US
```

Comment:
This is definitely debatable. I opted for `US` because the code pretty much didn't needed any changes. But `UK` is still an option.

Reply:
Does using default setting make sense? Default setting is more loose than `US` or `UK`.

https://github.com/client9/misspell#usage
```
-locale string
    	Correct spellings using locale perferances for US or UK.  Default is to use a neutral variety of English.  Setting locale to US will correct the British spelling of 'colour' to 'color'
```

If we have to chose one of `US` or `UK`, I prefer `US`. 0/5

Reply:
Let's go with `US`.